src/components/Button/index.js

```diff
@@ -4,14 +4,19 @@ import classnames from 'classnames'
 
 import styles from './styles.css'
 
-const Button = (props) => (
+const Button = ({ primary, ghost, secondary, size, block, ...props }) => (
   <button
     className={classnames(styles.btn, {
-      [styles.primary]: props.primary,
-      [styles.ghost]: props.ghost,
-      [styles.secondary]: props.secondary,
+      [styles.primary]: primary,
+      [styles.ghost]: ghost,
+      [styles.secondary]: secondary,
+      [styles.small]: size === 'small',
+      [styles.large]: size === 'large',
+      [styles.medium]: size === 'medium',
+      [styles.xlarge]: size === 'xlarge',
```

Comment:
What do you think of making these size props a boolean, similar to how `primary`, `ghost` and `secondary` props? That way we can use the component like this:
```js
<Button primary small>Content</Button>
```